### PR TITLE
DM-43213: Fix broken logic introduced by DM-43141

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: python/lsst/rubintv/production/starTracker.py [hunk 1]
```diff
@@ -873,6 +878,7 @@ def catchupCamera(self, camera, dayObs):
                                                 cameraType=camera.cameraType)
         for file in filenames:
             self.log.info(f'Catching up {file}')
+            gc.collect()
```
Comment (isullivan) [inline python/lsst/rubintv/production/starTracker.py:881]: Is this still necessary, after the matplotlib `plot` changes?

File: python/lsst/rubintv/production/starTracker.py [hunk 2]
```diff
@@ -399,6 +401,7 @@ def __init__(self,
         self.solver = CommandLineSolver(indexFilePath=self.locationConfig.astrometryNetRefCatPath,
                                         checkInParallel=True,
                                         timeout=30)
+        self.fig = plt.figure(figsize=(16, 16))
```
Comment (isullivan) [inline python/lsst/rubintv/production/starTracker.py:404]: Is there ever a case where you would want to pass an existing figure reference into this class?

File: python/lsst/rubintv/production/starTracker.py [hunk 3]
```diff
@@ -50,6 +51,7 @@
 from .uploaders import Uploader, Heartbeater
 from .baseChannels import BaseChannel
 from .plotting import starTrackerNightReportPlots
+import gc
```
Comment (isullivan) [inline python/lsst/rubintv/production/starTracker.py:54]: If you do still need this, move it up to the first block of imports.

**Review by isullivan — APPROVED** [review]
Looks good!

File: python/lsst/rubintv/production/starTracker.py (hunk 1 above)
Comment (mfisherlevine) [reply python/lsst/rubintv/production/starTracker.py:881]: You know, probably not, but I don't think it does any harm. Then again, I do dislike code which scatters those around for no reason, so maybe I will removed and see if it's all OK with it actually, I can always put it back, but if I leave it it'll likely live on forever.

File: python/lsst/rubintv/production/starTracker.py (hunk 2 above)
Comment (mfisherlevine) [reply python/lsst/rubintv/production/starTracker.py:404]: No that I can think of tbh.

**Pull request merged**